Description: Remove test_exclude_patterns test
 Under the sbuild environment, the asert doesn't work and sys.prefix gets
 wrong. So I'm just removing this test.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2022-09-14

--- python-werkzeug-2.2.2.orig/tests/test_serving.py
+++ python-werkzeug-2.2.2/tests/test_serving.py
@@ -125,16 +125,6 @@ def test_windows_get_args_for_reloading(
     assert rv == argv
 
 
-@pytest.mark.parametrize("find", [_find_stat_paths, _find_watchdog_paths])
-def test_exclude_patterns(find):
-    # Imported paths under sys.prefix will be included by default.
-    paths = find(set(), set())
-    assert any(p.startswith(sys.prefix) for p in paths)
-    # Those paths should be excluded due to the pattern.
-    paths = find(set(), {f"{sys.prefix}*"})
-    assert not any(p.startswith(sys.prefix) for p in paths)
-
-
 @pytest.mark.filterwarnings("ignore::pytest.PytestUnraisableExceptionWarning")
 @pytest.mark.dev_server
 def test_wrong_protocol(standard_app):
